SPECS/flannel/flannel.spec

```diff
@@ -0,0 +1,63 @@
+%global debug_package %{nil}
+%define our_gopath %{_topdir}/.gopath
+%define gopath_flannel_folder %{our_gopath}/src/github.com/coreos/flannel
+
+Summary:        Simple and easy way to configure a layer 3 network fabric designed for Kubernetes
+Name:           flannel
+Version:        0.12.0
+Release:        1%{?dist}
+License:        Apache License 2.0
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+Group:          System Environment/Libraries
+URL:            https://github.com/coreos/flannel
+#Source0:       https://github.com/coreos/flannel/archive/v0.12.0.tar.gz
+Source0:        %{name}-%{version}.tar.gz
+Patch0:         buid-no-static-glibc.patch
+
+BuildRequires:  golang >= 1.10.3
+
+%description
+Flannel is a simple and easy way to configure a layer 3 network fabric designed for Kubernetes.
+
+%prep
+%setup -q
+%patch0 -p1
+
+%build
+# move sources where make expect them to be (see flannel building.md)
+export GOPATH=%{our_gopath}
+mkdir -p "%{gopath_flannel_folder}"
+mv %{_builddir}/%{name}-%{version}/* %{gopath_flannel_folder}
+# but keep license file where it was to make %license macro happy
+cp %{gopath_flannel_folder}/LICENSE %{_builddir}/%{name}-%{version}
+
+# build flannel
+export TAG=v%{version}
+%ifarch x86_64
```

Comment:
Should we add some directives to mention that these two are the only valid architectures(e.g. BuildArch)?

Reply:
keep as it is since CBL-Mariner is only AMD64 and ARM64